[2023-06-01T21:10:49.867+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: etl_pj_br.DADOS_PRD.extract_data manual__2023-06-01T21:10:47.357189+00:00 [queued]>
[2023-06-01T21:10:49.891+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: etl_pj_br.DADOS_PRD.extract_data manual__2023-06-01T21:10:47.357189+00:00 [queued]>
[2023-06-01T21:10:49.891+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T21:10:49.891+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2023-06-01T21:10:49.892+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T21:10:49.917+0000] {taskinstance.py:1304} INFO - Executing <Task(_PythonDecoratedOperator): DADOS_PRD.extract_data> on 2023-06-01 21:10:47.357189+00:00
[2023-06-01T21:10:49.926+0000] {standard_task_runner.py:55} INFO - Started process 7063 to run task
[2023-06-01T21:10:49.930+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'etl_pj_br', 'DADOS_PRD.extract_data', 'manual__2023-06-01T21:10:47.357189+00:00', '--job-id', '84', '--raw', '--subdir', 'DAGS_FOLDER/pip_example.py', '--cfg-path', '/tmp/tmpc3qb0esu']
[2023-06-01T21:10:49.932+0000] {standard_task_runner.py:83} INFO - Job 84: Subtask DADOS_PRD.extract_data
[2023-06-01T21:10:50.013+0000] {task_command.py:389} INFO - Running <TaskInstance: etl_pj_br.DADOS_PRD.extract_data manual__2023-06-01T21:10:47.357189+00:00 [running]> on host e77927a9666c
[2023-06-01T21:10:50.114+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_pj_br
AIRFLOW_CTX_TASK_ID=DADOS_PRD.extract_data
AIRFLOW_CTX_EXECUTION_DATE=2023-06-01T21:10:47.357189+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-01T21:10:47.357189+00:00
[2023-06-01T21:10:50.115+0000] {logging_mixin.py:137} INFO - Reading file...
[2023-06-01T21:10:50.115+0000] {logging_mixin.py:137} INFO - {'conf': <***.configuration.AirflowConfigParser object at 0x7fd915f19290>, 'dag': <DAG: etl_pj_br>, 'dag_run': <DagRun etl_pj_br @ 2023-06-01 21:10:47.357189+00:00: manual__2023-06-01T21:10:47.357189+00:00, state:running, queued_at: 2023-06-01 21:10:47.368437+00:00. externally triggered: True>, 'data_interval_end': DateTime(2023, 6, 1, 21, 10, 47, 357189, tzinfo=Timezone('UTC')), 'data_interval_start': DateTime(2023, 6, 1, 21, 10, 47, 357189, tzinfo=Timezone('UTC')), 'ds': '2023-06-01', 'ds_nodash': '20230601', 'execution_date': <Proxy at 0x7fd8ebd9b690 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x7fd8ebd2da70>, 'execution_date', DateTime(2023, 6, 1, 21, 10, 47, 357189, tzinfo=Timezone('UTC')))>, 'expanded_ti_count': None, 'inlets': [], 'logical_date': DateTime(2023, 6, 1, 21, 10, 47, 357189, tzinfo=Timezone('UTC')), 'macros': <module '***.macros' from '/home/***/.local/lib/python3.7/site-packages/***/macros/__init__.py'>, 'next_ds': <Proxy at 0x7fd8ebd9bc80 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x7fd8ebd2da70>, 'next_ds', '2023-06-01')>, 'next_ds_nodash': <Proxy at 0x7fd8ebdaa5f0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x7fd8ebd2da70>, 'next_ds_nodash', '20230601')>, 'next_execution_date': <Proxy at 0x7fd8ebdaa550 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x7fd8ebd2da70>, 'next_execution_date', DateTime(2023, 6, 1, 21, 10, 47, 357189, tzinfo=Timezone('UTC')))>, 'outlets': [], 'params': {}, 'prev_data_interval_start_success': None, 'prev_data_interval_end_success': None, 'prev_ds': <Proxy at 0x7fd8ebdaa5a0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x7fd8ebd2da70>, 'prev_ds', '2023-06-01')>, 'prev_ds_nodash': <Proxy at 0x7fd8ebde3c80 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x7fd8ebd2da70>, 'prev_ds_nodash', '20230601')>, 'prev_execution_date': <Proxy at 0x7fd8ebde3eb0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x7fd8ebd2da70>, 'prev_execution_date', DateTime(2023, 6, 1, 21, 10, 47, 357189, tzinfo=Timezone('UTC')))>, 'prev_execution_date_success': <Proxy at 0x7fd8ebde6190 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x7fd8ebd2da70>, 'prev_execution_date_success', None)>, 'prev_start_date_success': None, 'run_id': 'manual__2023-06-01T21:10:47.357189+00:00', 'task': <Task(_PythonDecoratedOperator): DADOS_PRD.extract_data>, 'task_instance': <TaskInstance: etl_pj_br.DADOS_PRD.extract_data manual__2023-06-01T21:10:47.357189+00:00 [running]>, 'task_instance_key_str': 'etl_pj_br__DADOS_PRD.extract_data__20230601', 'test_mode': False, 'ti': <TaskInstance: etl_pj_br.DADOS_PRD.extract_data manual__2023-06-01T21:10:47.357189+00:00 [running]>, 'tomorrow_ds': <Proxy at 0x7fd8ebd712d0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x7fd8ebd2da70>, 'tomorrow_ds', '2023-06-02')>, 'tomorrow_ds_nodash': <Proxy at 0x7fd8ebd71780 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x7fd8ebd2da70>, 'tomorrow_ds_nodash', '20230602')>, 'triggering_dataset_events': <Proxy at 0x7fd8ebdd7d20 with factory <function TaskInstance.get_template_context.<locals>.get_triggering_events at 0x7fd8ebde57a0>>, 'ts': '2023-06-01T21:10:47.357189+00:00', 'ts_nodash': '20230601T211047', 'ts_nodash_with_tz': '20230601T211047.357189+0000', 'var': {'json': None, 'value': None}, 'conn': None, 'yesterday_ds': <Proxy at 0x7fd8ebd71230 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x7fd8ebd2da70>, 'yesterday_ds', '2023-05-31')>, 'yesterday_ds_nodash': <Proxy at 0x7fd8ebd98c30 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x7fd8ebd2da70>, 'yesterday_ds_nodash', '20230531')>, 'templates_dict': None}
[2023-06-01T21:10:50.119+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 217, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/pip_example.py", line 25, in extract
    df = pd.read_csv('./data/gov.csv', sep=';', encoding='latin', names=['code', 'name'])
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/util/_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 586, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 482, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 811, in __init__
    self._engine = self._make_engine(self.engine)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 1040, in _make_engine
    return mapping[engine](self.f, **self.options)  # type: ignore[call-arg]
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 51, in __init__
    self._open_handles(src, kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/base_parser.py", line 229, in _open_handles
    errors=kwds.get("encoding_errors", "strict"),
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/common.py", line 707, in get_handle
    newline="",
FileNotFoundError: [Errno 2] No such file or directory: './data/gov.csv'
[2023-06-01T21:10:50.133+0000] {taskinstance.py:1327} INFO - Marking task as FAILED. dag_id=etl_pj_br, task_id=DADOS_PRD.extract_data, execution_date=20230601T211047, start_date=20230601T211049, end_date=20230601T211050
[2023-06-01T21:10:50.150+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 84 for task DADOS_PRD.extract_data ([Errno 2] No such file or directory: './data/gov.csv'; 7063)
[2023-06-01T21:10:50.181+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-06-01T21:10:50.229+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
